src/frontend/src/views/Privacy/index.tsx

```diff
@@ -31,17 +31,17 @@ const PrivacyPolicy = (): JSX.Element => {
         <H4>Last Updated: May 17, 2021. </H4>
       </Title>
       <P>
-        The Chan Zuckerberg Initiative Foundation, a 501(c)(3) nonprofit private
```

Comment:
Are you looking at a different doc than I am? The version I'm looking at (from the Shortcut story) still has the part about "501(c)(3) nonprofit private foundation". Are we looking at different Privacy Policy docs?